app/src/main/java/org/simple/clinic/teleconsultlog/success/TeleConsultSuccessEffectHandler.kt

```diff
@@ -27,6 +28,7 @@ class TeleConsultSuccessEffectHandler @AssistedInject constructor(
         .subtypeEffectHandler<TeleConsultSuccessEffect, TeleConsultSuccessEvent>()
         .addTransformer(LoadPatientDetails::class.java, loadPatientDetails(schedulersProvider.io()))
         .addConsumer(GoToHomeScreen::class.java, { uiActions.goToHomeScreen() }, schedulersProvider.ui())
+        .addConsumer(GoToPrescriptionScreen::class.java, { uiActions.goToPrescriptionScreen() }, schedulersProvider.ui())
```

Comment:
Same comment as earlier about this being an action instead of a consumer.

Reply:
This would be `addConsumer` since we're sending `patientUuid` to Prescription screen